### ISSUE TITLE
Module not found error

### ISSUE THREAD

**Issue by chuyaowang** [whitphx/stlite#558]
Hi, I followed your guide on converting a streamlit app to electron. After I run `npm run serve`, this error shows up:

`ModuleNotFoundError: The module 'distutils' is unvendored from the Python standard library in the Pyodide distribution. You can install it by calling: await micropip.install("distutils") in Python, or await pyodide.loadPackage("distutils") in JavaScript See https://pyodide.org/en/stable/usage/loading-packages.html for more details.`

Here is the traceback:

```
File "/lib/python3.10/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 564, in _run_script
    exec(code, module.__dict__)
File "/home/pyodide/streamlit_app.py", line 2, in <module>
    from sidebar import sideBar
File "/home/pyodide/sidebar.py", line 6, in <module>
    from joblib import load, dump
File "/lib/python3.10/site-packages/joblib/__init__.py", line 113, in <module>
    from .memory import Memory, MemorizedResult, register_store_backend
File "/lib/python3.10/site-packages/joblib/memory.py", line 32, in <module>
    from ._store_backends import StoreBackendBase, FileSystemStoreBackend
File "/lib/python3.10/site-packages/joblib/_store_backends.py", line 15, in <module>
    from .backports import concurrency_safe_rename
File "/lib/python3.10/site-packages/joblib/backports.py", line 22, in <module>
    import distutils  # noqa
```

I tried to add distutils to my requirements.txt, but it is still not installed. Where would I run the `await micropip.install("distutils")` or `await pyodide.loadPackage("distutils")` command?

**Comment by chuyaowang** [comment]
Here is my repo: https://github.com/chuyaowang/rfApp

**Comment by whitphx** [comment]
Sorry, it's a lack of the stlite's capability.
This problem will be tracked in #564 .

**Comment by whitphx** [comment]
This should be fixed with v0.33.0.
Please try it. Thanks.
